### PR TITLE
Tags provider class

### PR DESCRIPTION
**Pull request by Slickoo**
Class for getting tags from audio.

### CODE REVIEW

File: NPlaylist/Tests/NPlaylist.Business.Tests/MetaTags/TagLibTagsProviderTests.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿using FluentAssertions;
+using NPlaylist.Business.MetaTags;
+using System;
+using Xunit;
+
+namespace NPlaylist.Business.Tests.MetaTags
+{
+    public class TagLibTagsProviderTests
+    {
```
Comment (Dorinadumbrava) [inline NPlaylist/Tests/NPlaylist.Business.Tests/MetaTags/TagLibTagsProviderTests.cs]: I would also test that i get the exception for the empty file path
Also, I would test the return value : GetTags_ReturnsCorrectAudioMeta_True

File: NPlaylist/NPlaylist.Business/MetaTags/TagLibTagsProvider.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        {
+            _tagLibWrapper = tagLibWrapper;
+        }
+
+        public AudioMeta GetTags(string uploadedFilePath)
+        {
+            if (string.IsNullOrWhiteSpace(uploadedFilePath))
+            {
+                throw new ArgumentException(nameof(uploadedFilePath));
+            }
```
Comment (Dorinadumbrava) [inline NPlaylist/NPlaylist.Business/MetaTags/TagLibTagsProvider.cs]: I would add validation for extension/format of the filePath as well
